hw/ip/otbn/data/bignum-insns.yml

```diff
@@ -267,12 +258,14 @@
     - *mulqacc-wrs2
     - *mulqacc-wrs2-qwsel
     - *mulqacc-acc-shift-imm
+    - *bn-flag-group-operand
   syntax: |
-    [<zero_acc>] <wrd>, <wrs1>.<wrs1_qwsel>, <wrs2>.<wrs2_qwsel>, <acc_shift_imm>
+    [<zero_acc>] <wrd>, <wrs1>.<wrs1_qwsel>, <wrs2>.<wrs2_qwsel>, <acc_shift_imm>[, FG<flag_group>]
   glued-ops: true
   doc: |
-    Multiplies two `WLEN/4` WDR values, shifts the product by `acc_shift_imm` bit, and adds the result to the accumulator.
+    Multiplies two `WLEN/4` WDR values, shifts the product by `acc_shift_imm` bits, and adds the result to the accumulator.
     Writes the resulting accumulator to `wrd`.
+    This updates the M, L and Z flags of `flag_group`.
   decode: |
     writeback_variant = 'writeout'
```

Comment:
If there any need for the `writeback_variant` in decode now it's not being used in the operation pseudo code?

Reply:
Good point, no! I've got rid of it.